falcon/vendor/mimeparse/mimeparse.py

```diff
@@ -31,8 +31,7 @@ def parse_mime_type(mime_type):
 
     type_parts = full_type.split('/') if '/' in full_type else None
     if not type_parts or len(type_parts) > 2:
-        raise MimeTypeParseException(
-            "Can't parse type \"{}\"".format(full_type))
+        raise MimeTypeParseException("Can't parse type \"{}\"".format(full_type))
```

Comment:
Maybe we should exclude vendored modules from blackening?

It might make it more cumbersome to backport changes/sync with upstream, even though any changes are looking unlikely at this point.

Reply:
oh good point. let me try to exclude them.